hr_employee_lastnames/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Could you use the new OCA fashion about Readme folder?
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

A bot auto-build this README.rst file using the folder one
Regards

Reply:
Forget it

The folder exists